core/modules/views/views.install

```diff
@@ -192,6 +192,29 @@ function views_update_1007() {
   config_clear('views.settings', 'no_javascript');
 }
 
+/**
+ * Set all grid style Views to use the backwards-compatible unresponsive option
+ * if not already set.
+ */
+function views_update_1008() {
+  $view_names = config_get_names_with_prefix('views.view.');
+  foreach ($view_names as $view_name) {
+    $config = config($view_name);
+    $displays = $config->get('display');
+    foreach ($displays as $display => $display_details) {
+      if (!isset($display_details['display_options']['style_plugin']) || $display_details['display_options']['style_plugin'] != 'grid') {
+        continue;
+      }
+      if (!empty($display_details['display_options']['style_options']['unresponsive'])) {
```

Comment:
I think this should be `isset()`, so that if this update hook is run after views have been built with the new option, they keep their set option, rather than forcing to unresponsive.

```suggestion
      if (isset($display_details['display_options']['style_options']['unresponsive'])) {
```